system-tests/projects/no-specs-vue-2/.gitignore

```diff
@@ -0,0 +1,23 @@
+.DS_Store
```

Comment:
We should remove any extraneous files from this system-test project, like .gitignore and README.md. Might be able to trim it down further by removing the babel and eslint depednencies/configurations.